### PR TITLE
created playlist page

### PR DESCRIPTION
**Pull request by Rockyraven**
feat: 
created Model to add playlist 
some video image added in backend
created playlist page
some CSS updated

### CODE REVIEW

File: src/context/libraryContext.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      const response = await axios.post(
+        `/api/user/playlists/${id}`,
+        { video },
+        {
+          headers: {
+            authorization: user.encodedToken,
+          },
+        }
+        );
+        // setLibrary(response.data.playlists)
```
Comment (swethaogeti) [inline src/context/libraryContext.js]: remove comments

**Review by swethaogeti — APPROVED** [review]
code looks clean and readiable , in one of the doc it is advised not to store the user data in local
storage . pls do check . Rest is good .GTM.

**Comment by Rockyraven** [comment]
> code looks clean and readiable , in one of the doc it is advised not to store the user data in local storage . pls do check . Rest is good .GTM.


ok I will do Changes

Thanku for review

**Review by Shubhamkr2610 — APPROVED** [review]
after removing  some unused code , then you are good to go .

**Pull request merged**